### PR TITLE
observability/mixins: Add Makefile step that generates dashboards

### PR DESCRIPTION
**Pull request by ArthurSens**
Signed-off-by: ArthurSens <arthursens2005@gmail.com>

## Description

Just a small improvement to the DevX when developing mixins with Jsonnet. 

When getting started it's hard to understand how all the imports and dependencies work, so I'm just adding a Makefile step that is able to generate the JSON file for all dashboards included in our mixins. Useful to make sure we've imported dashboards objects to the right place

cc @sagor999 

## Related Issue(s)

Related to https://github.com/gitpod-io/gitpod/pull/7966#issuecomment-1027778859

## How to test

```shell
cd operations/observability/mixins
make dashboards_out
```

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **arthursens** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[operations/observability/mixins/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by wulfthimm — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: edf3d46ba7cf59cb6a5d14cbb91ad806f6cd8cff</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7972#pullrequestreview-870341481" title="Approved">wulfthimm</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[operations/observability/mixins/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/OWNERS)~~ [wulfthimm]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7972#pullrequestreview-870341481" title="Approved">wulfthimm</a>*

*No associated issue*. Requirement bypassed by manually added approval.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[operations/observability/mixins/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/OWNERS)~~ [wulfthimm]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
